doc-examples/example-java/build.gradle.kts

```diff
@@ -22,3 +22,10 @@ dependencies {
 application {
     mainClass.set("example.Application")
 }
+
+//tasks {
```

Comment:
did you leave this intentionally for docs?

Reply:
No, just in case I needed it again I left it there :)